templates/zerver/help/delete-a-stream.md

```diff
@@ -1,29 +1,20 @@
 # Delete a stream
 
-In Zulip, most of the stream administration is done on the
-subscription page.  Nevertheless, that is not the case for deleting a
-stream.  Follow these instructions to learn how it is done.
+In Zulip, most of the stream administration is done on the subscription page.
+However, Zulip organization administrators must use their administration panels
+to delete streams.
 
-## Locate the administration tab
+{!go-to-the.md!} [Streams Deletion](/#administration/streams-list-admin)
```

Comment:
Should this be "Streams Deletion" or "Stream Deletion"? not entirely sure. plural sounds a little awkward.